test-network-function/diagnostic/suite.go

```diff
@@ -112,6 +125,18 @@ type NodesHwInfo struct {
 	Worker NodeHwInfo
 }
 
+// CsiDriver stores csi driver JSON output from  `oc get csidirver -o json` command
+type CsiDriver struct {
+	Kind     string `json:"kind"`
```

Comment:
what is the point of this field if it's always the same value?

Reply:
just want to specifiy in the claim file that this part is CSI Driver info